src/src/PluginStructs/P099_data_struct.cpp

```diff
@@ -252,7 +269,7 @@ void P099_data_struct::scaleRawToCalibrated(uint16_t &x, uint16_t &y) {
       if (_x > StoredSettings.Calibration.bottom_right.x) {
         _x = StoredSettings.Calibration.bottom_right.x;
       }
-      float x_fact = (StoredSettings.Calibration.bottom_right.x - StoredSettings.Calibration.top_left.x) / ts_x_res;
+      float x_fact = (StoredSettings.Calibration.bottom_right.x - StoredSettings.Calibration.top_left.x) / _ts_x_res;
       x = int((_x * 1.0f) / x_fact);

```

Comment:
Multiplying by `1.0f` is not really needed here as you divide it by a float (which may not be needed, see other comment on `y_fact`)
If you need to cast it to a float, use `static_cast<float>(_x)`